app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/MainFragment.kt

```diff
@@ -95,6 +100,19 @@ class MainFragment : BaseFragment<Unit>(
         viewModel.cards.observe(viewLifecycleOwner) {
             it?.let { onNewCardsAvailable(it) }
         }
+
+        // Triggers a Google Play store review if the user has experienced some key features of the app
+        // The earliest possible re-trigger of a review prompt occurs after half a year
+        val dateFormatter = SimpleDateFormat("MM-yyyy", Locale.GERMANY)
```

Comment:
Unless I'm missing something, instead of serializing and parsing the last review date you can just save `Date().time` (the timestamp in milliseconds) and compare it to the current time, right?

Reply:
BTW, you can also use `org.joda.time.*`, it has a few fancy methods that could be handy like `DateTime.now().minusMonths(6).isBefore(...)`

Reply:
Thanks for that tip, that does indeed make the code a lot more readable and also allows removal of that ugly constant. I wasn't aware of that date helper class.